### PR TITLE
Modern ui

### LINKED ISSUE #15: Store all the visit to better display in clients

### PR DESCRIPTION
**Pull request by erwannT**
Fix #15 
Update UI

### CODE REVIEW

**Comment by ggerbaud** [comment]
Wouldn't it be easier to store the visit in the GameState (and remove LastSector) directly instead of PlayerState, since it is reset each time we change player ?

File: webapp/src/NewGameButton.js [hunk 1]
```diff
@@ -14,17 +14,21 @@ class NewGameButton extends Component {
...
-     <span>{this.props.flavor.Code}</span>
-     <a onClick={() => this.newGame(this.props.flavor.Code)} className="secondary-content"><i className="material-icons light-blue-text">keyboard_arrow_right</i></a>
-     </li>)
+    return (
+      <li>
+        <div className="collapsible-header">
+          <span>{this.props.flavor.Code}</span>
+          <a onClick={() => this.newGame(this.props.flavor.Code) } className="secondary-content"><i className="material-icons light-blue-text">keyboard_arrow_right</i></a>
+        </div>
+        <div className="collapsible-body"><p>Voluptate ea aliquip esse consequat eu reprehenderit laborum sunt sit. Esse labore duis amet sint in veniam aute esse enim. Adipisicing culpa quis aliqua est excepteur magna. Nostrud amet incididunt irure duis ea exercitation qui. Est pariatur est pariatur non pariatur anim dolore velit reprehenderit commodo id consequat proident.</p></div>
```
Comment (ggerbaud) [inline webapp/src/NewGameButton.js]: je pense que c'est plutot this.props.flavor.Desc ici

File: webapp/src/NewPlayerButton.js [hunk 2]
```diff
@@ -11,26 +26,41 @@ class NewPlayerButton extends Component {
...
       },
       body: JSON.stringify({
-        name: this.refs.name.value,
-        board: "Rennes"
+        name: this._nameinput.value,
+        board: this._boardinput.selectInput.value
       })
     })
       .then((response) => response.json())
       .then((json) => console.log(json))
```
Comment (ggerbaud) [inline webapp/src/NewPlayerButton.js]: We should erase playerName after a successful add

File: webapp/src/NextPlayerButton.js [hunk 3]
```diff
@@ -17,9 +17,10 @@ class NextPlayerButton extends Component {
     }
 
     render() {
+        const label = ( this.props.game.Ongoing == 3 ? "Hold" : "Next" )
```
Comment (ggerbaud) [inline webapp/src/NextPlayerButton.js]: This should be inverted. It should display 'Hold' when playing and 'Next' while on hold

File: webapp/src/App.js [hunk 4]
```diff
@@ -14,34 +14,36 @@ class App extends Component {
...
+            <img  src={logo} className="App-logo" alt="logo" />
+          </div>
+          <h4 className="center header-block">Welcome to godart</h4>
+
+          <div className="col s12 m4" >
             <div className="icon-block">
-              <h2 className="center light-blue-text"><i className="material-icons">add_circle</i></h2>
               <div className="center">
-                <Link to="/newGame" className="btn-large waves-effect waves-light orange">Get started</Link>
+                <Link to="/newGame" className="btn-large waves-effect waves-light btn"><i className="material-icons left large ">add</i>New Game</Link>
```
Comment (ggerbaud) [inline webapp/src/App.js]: It should be light-blue or all the light-blue elements should be green as this one. But we shouldn't mix colors

**Comment by ggerbaud** [comment]
Looks really good to me, just few things to change for me

File: webapp/src/NewGameButton.js (hunk 1 above)
Comment (erwannT) [inline webapp/src/NewGameButton.js]: Yes, i didn't see desc on 'flavor' object. 
A simple string may be not sufficient for a good description of each game.

Comment (ggerbaud) [inline webapp/src/NewGameButton.js]: You mean `string` is not a good fit to store the game description or the actual description are not long enough ?

Comment (erwannT) [inline webapp/src/NewGameButton.js]: For me, description is  game rule.

Comment (ggerbaud) [inline webapp/src/NewGameButton.js]: So we'll have to improve it here https://github.com/gocaine/go-dart/blob/master/common/structs.go#L105

**Comment by mathieu-pousse** [comment]
lgtm, impressive !

**Comment by erwannT** [comment]
@ggerbaud @mathieu-pousse 
My branch is rebased. 
I fix color probleme and reversal between "next" and "hold" button
Can you LGTM again?

**Comment by ggerbaud** [comment]
Game description is still a lorem ipsum like

**Comment by erwannT** [comment]
I will add an issue for this problem.

**Comment by ggerbaud** [comment]
Well the issue exists #46 
I just though you replace it the sample text by the real property in this PR

LGTM thus

**Comment by mathieu-pousse** [comment]
LGTM

**Pull request merged**